app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -17,7 +17,7 @@ def import_proxy_select(hash)
 
   def import_proxy_links(hash, classes = nil)
     SmartProxy.with_features("Puppet").map do |proxy|
-      display_link_if_authorized(_("Import from %s") % proxy.name, hash.merge(:proxy => proxy), {:class=>classes, :"data-no-turbolink" => true})
+      display_link_if_authorized(_("Import from %s") % proxy.name, hash.merge(:proxy => proxy), {:class=>classes})
```

Comment:
this change is what caused me to replace the progress bar, but it's seemingly unrelated to the task description, should I move it to a new PR?